composer.json

```diff
@@ -58,7 +60,7 @@
             "vendor/bin/phpunit --verbose --configuration phpunit.xml.dist --coverage-html tests/html-report"
         ],
         "phpcs": [
-            "vendor/bin/php-cs-fixer fix --config=.php_cs --verbose --diff --dry-run"
+            "vendor/bin/php-cs-fixer fix --config=.php_cs.dist --verbose --diff --dry-run"
```

Comment:
Maybe we could omit the `--config` altogether? IIRC it chooses the right one by default.